packages/core/integration-tests/test/javascript.js

```diff
@@ -662,7 +662,8 @@ describe('javascript', function() {
 
   it('should support importing a URL to a raw asset', async function() {
     let b = await bundle(
-      path.join(__dirname, '/integration/import-raw/index.js')
+      path.join(__dirname, '/integration/import-raw/index.js'),
+      {disableCache: false}
```

Comment:
hmm, not sure about changing this for the tests. why did you need this change?

Reply:
@devongovett It's to turn on caching to simulate the asset being written to cache, in order to get the number of bytes written to disk (to prove that it's not 0)